pkg/sql/catalog/table_elements.go

```diff
@@ -400,6 +424,7 @@ type Column interface {
 // Constraint is an interface around a constraint.
 type Constraint interface {
```

Comment:
Why do we also want to introduce these two methods: `IsConstraintValidated` and `IsConstraintUnvalidated` when we can just check `GetConstraintValidity`?

In fact, why do we even need `IsEnforced`? Can we just say that `GetConstraintValidity` is all we need, bc
- Validating, Validated, Unvalidated ==> enforced
- Dropping ==> not enforced

Reply:
Dropping constraints are enforced. At least, that's the case of unique indexes. You can see this for yourself by setting a pausepoint on the schema changer execution, dropping a unique index, and INSERTing two identical values.